### PR TITLE
Issue 687 686 new webconsole tests

### PR DESCRIPTION
**Pull request by kornys**

### CODE REVIEW

File: systemtests/src/test/java/io/enmasse/systemtest/web/WebConsoleTest.java [hunk 1]
```diff
@@ -303,6 +304,46 @@ public void doTestSortConnectionsByHostname() throws Exception {
...
+        Destination dest = Destination.queue("queue-via-web");
+        consoleWebPage.createAddressWebConsole(dest);
+        consoleWebPage.openAddressesPageWebConsole();
+
+        AmqpClient client = amqpClientFactory.createQueueClient(defaultAddressSpace);
+        client.getConnectOptions().setUsername(username).setPassword(password);
+        List<String> msgBatch = TestUtils.generateMessages(msgCount);
+
+        int sent = client.sendMessages(dest.getAddress(), msgBatch, 1, TimeUnit.MINUTES).get(1, TimeUnit.MINUTES);
+        Thread.sleep(5000);
```
Comment (tkratky) [inline systemtests/src/test/java/io/enmasse/systemtest/web/WebConsoleTest.java]: This sleep could be replace by some wait until expected count of messages/senders/receivers/...etc within some timeout (i.e 5sec)

File: systemtests/src/test/java/io/enmasse/systemtest/web/WebConsoleTest.java [hunk 2]
```diff
@@ -303,6 +304,46 @@ public void doTestSortConnectionsByHostname() throws Exception {
...
+        assertEquals(sent, consoleWebPage.getAddressItem(dest).getMessagesIn());
+        assertEquals(msgCount, consoleWebPage.getAddressItem(dest).getMessagesStored());
+
+        int received = client.recvMessages(dest.getAddress(), msgCount).get(1, TimeUnit.MINUTES).size();
+        Thread.sleep(5000);
```
Comment (tkratky) [inline systemtests/src/test/java/io/enmasse/systemtest/web/WebConsoleTest.java]: the same as above

File: systemtests/src/test/java/io/enmasse/systemtest/web/WebConsoleTest.java [hunk 3]
```diff
@@ -303,6 +304,46 @@ public void doTestSortConnectionsByHostname() throws Exception {
...
+        int senderCount = 5;
+        int receiverCount = 10;
+        Destination dest = Destination.queue("queue-via-web");
+        consoleWebPage.createAddressWebConsole(dest);
+        consoleWebPage.openAddressesPageWebConsole();
+
+        AbstractClient client = new RheaClientConnector();
+        try{
+            client = attachConnector(dest, 1, senderCount, receiverCount);
+            Thread.sleep(5000);
```
Comment (tkratky) [inline systemtests/src/test/java/io/enmasse/systemtest/web/WebConsoleTest.java]: the same as above

File: systemtests/src/test/java/io/enmasse/systemtest/web/WebConsoleTest.java (hunk 1 above)
Comment (kornys) [reply systemtests/src/test/java/io/enmasse/systemtest/web/WebConsoleTest.java]: resolved

**Pull request merged**